### PR TITLE
Barebones admin dash

### PR DESCRIPTION
**Pull request by Sonjajo**
## Barebones Admin Review Dashboard

_This is where admins will be able to review organizations. Based off Elizabeths most recent mock-up, I created an OrgCard component and added some test data in sample-data.ts. Each OrgCard has a logo, name, date, and description. The admin index page accesses the OrgCards through a separate OrgList component. Each Org also has it's own page, generated by OrgDetail._

_I began to try to implement a css module but need some help with syntax._

## Screenshots
This is what the admin dash looks like now. There's example data for two organizations. 
image

CC: @kwfk

### CODE REVIEW

File: components/OrgCard.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
+// import Head from 'next/head';
+import { Org } from 'interfaces';
+// import styles from 'styles/OrgCard.module.css';
```
Comment (kwfk) [inline components/OrgCard.tsx]: let's remove the comments for cleaner code 😆

File: components/OrgCard.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+type Props = {
+  data: Org;
+};
+
+// <OrgCard className={styles.OrgCard}> OrgCard</OrgCard>
```
Comment (kwfk) [inline components/OrgCard.tsx]: here too comments

File: pages/admin/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import Layout from 'components/Layout';
+import List from 'components/OrgList';
+
+type Props = {
+  items: Org[];
+};
+
+// TODO: put the components on line 24, where items currently is
+
+const WithStaticProps: React.FunctionComponent<Props> = ({ items }) => (
```
Comment (kwfk) [inline pages/admin/index.tsx]: rename component to `ModeratorList`

File: pages/admin/index.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+export const getStaticProps: GetStaticProps = async () => {
+  // Example for including static props in a Next.js function component page.
+  // Don't forget to include the respective types for any props passed into
+  // the component.
+  const items: Org[] = sampleOrgData;
+  return { props: { items } };
+};
+
+export default WithStaticProps;
```
Comment (kwfk) [inline pages/admin/index.tsx]: `ModeratorList` here too

File: pages/admin/[id].tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+import { sampleOrgData } from 'utils/sample-data';
+import Layout from 'components/Layout';
+import OrgDetail from 'components/ListDetail';
+
+type Props = {
+  item?: Org;
+  errors?: string;
+};
+
+const StaticPropsDetail: React.FunctionComponent<Props> = ({
```
Comment (kwfk) [inline pages/admin/[id].tsx]: rename component to `ModeratorDashboard`

File: pages/admin/[id].tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+      title={`${
+        item ? item.name : 'Admin Detail'
+      } | Next.js + TypeScript Example`}
+    >
+      {item && <OrgDetail item={item} />}
+    </Layout>
+  );
+};
+
+export default StaticPropsDetail;
```
Comment (kwfk) [inline pages/admin/[id].tsx]: `ModeratorDashboard` here too

File: components/OrgList.tsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
+// TODO: instead of creating another list component, can I use the List.tsx file somehow?
```
Comment (kwfk) [inline components/OrgList.tsx]: let's merge in this in for now and you can change in your next PR

File: utils/sample-data.ts [hunk 8]
```diff
@@ -1,10 +1,34 @@
...
   { id: 104, name: 'Dave' },
 ];
+
+export const sampleAdminData: Admin[] = [
+  { id: 105, name: 'Sonja' },
+  { id: 106, name: 'Fred' },
+];
+
+// TODO: implement logo and date differently...
+export const sampleOrgData: Org[] = [
```
Comment (kwfk) [inline utils/sample-data.ts]: let's change this to `sampleOrgAppData`

**Review by kwfk — CHANGES_REQUESTED** [review]
let's change `admin` to `moderator` to reflect name change
you also want to rebase from main (you can follow the steps [here](https://calblueprint.slack.com/archives/C019DV6TSMA/p1601868963054400))

otherwise, looks good! good job on your first PR!

File: components/OrgCard.tsx [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import Link from 'next/link';
+import { Org } from 'interfaces';
```
Comment (kwfk) [inline components/OrgCard.tsx]: need to change all instances of `Org` interfaces to `Organization`

File: components/OrgList.tsx [hunk 10]
```diff
@@ -0,0 +1,18 @@
...
+import OrgCard from './OrgCard';
+
+type Props = {
+  items: Org[];
+};
+
+const List: React.FunctionComponent<Props> = ({ items }) => (
+  <>
+    {items.map((item) => (
+      <OrgCard data={item} />
```
Comment (kwfk) [inline components/OrgList.tsx]: need to add a `key` prop to items in a list
you can see this error in the browser console:
```
Warning: Each child in a list should have a unique "key" prop.
```

**Review by kwfk — CHANGES_REQUESTED** [review]
just some minor changes and you should be good!
nice work!

File: components/OrgCard.tsx (hunk 9 above)
Comment (Sonjajo) [reply components/OrgCard.tsx]: Ended up changing all instances to OrgApp.

**Review by kwfk — APPROVED** [review]
LGTM nice work

**Pull request merged**